uniswap-v2/src/pool_balance_retriever.rs

```diff
@@ -0,0 +1,51 @@
+use std::ops::Deref;
+
+use substreams::scalar::BigInt;
+use substreams::store::{DeltaBigInt, Deltas, StoreAdd, StoreAddBigInt};
+use substreams::Hex;
+use substreams_ethereum::NULL_ADDRESS;
+
+use crate::store_key::StoreKey;
+
+pub(crate) struct PoolBalanceRetriever<'a> {
```

Comment:
Why is this called `PoolBalanceRetriever`? Is it intended to be used to retrieve pool balance?